packages/cli/src/commands/dns/add.ts

```diff
@@ -35,6 +36,19 @@ export default async function add(
 
   const addStamp = stamp();
   const { domain, data: argData } = parsedParams;
+  const valueArgs = args.slice(3); // domain, name, type, ...valueArgs
```

Comment:
I looked into updating `parseAddDNSRecordArgs` to make this more directly available, but it gets complicated with (1) the different returns types and (2) passing this exact value to the API endpoint later in the flow.